hddfancontrol/__init__.py

```diff
@@ -1184,7 +1228,7 @@ def cl_main():  # noqa: C901
         type=int,
         nargs=2,
         default=(None, None),
-        help="""CPU temperature range, if CPU temp monitoring is enabled.
+        help="""CPU temperature range (max, critical), if CPU temp monitoring is enabled.
```

Comment:
This change seems unrelated with the rest of the pull request, please remove it.

Reply:
Yeah that's right, but it took me some time to determine what the two values are.
I will remove it from the PR but can you add this clarification as regular commit?

Reply:
Unless you have found a bug (in this case please open an issue), the temperature range set here should be the minimum and maximum temperature the user wants the device to stay in between.